configure.py

```diff
@@ -266,21 +255,22 @@ def _is_local(system):
 
 
 def _get_external_deps(args):
-    deps = ["nopemd"]
+    deps = [
+        "nopemd",
+        "glslang",
+    ]
```

Comment:
I guess this needs to drop the glslang references in `doc/usr/howto/installation.md`

Reply:
Done.